libcloud/compute/drivers/openstack.py

```diff
@@ -3160,6 +3160,12 @@ def _to_network(self, obj):
             extra["router:external"] = obj.get("router:external")
         if obj.get("subnets", None):
             extra["subnets"] = obj.get("subnets")
+        if obj.get("tags", None):
```

Comment:
Per codecov comment (https://github.com/apache/libcloud/pull/2016#issuecomment-2154326853) could you please add additional test cases so all the new code paths are covered?

This includes tests for all those fields being present and not being present (so both the explicit "if" and implicit "else" code branches are covered).

Thanks.

Reply:
Done!